app/app.config.js

```diff
@@ -0,0 +1,17 @@
+export default ({ config }) => {
```

Comment:
I think we should use Typescript version
https://docs.expo.dev/workflow/configuration/#using-typescript-for-configuration-appconfigts-instead-of